packages/jest-expo/src/enzyme/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = function withEnzyme(preset = {}) {
+  const { snapshotSerializers = [], haste = {}, setupFilesAfterEnv = [], setupFiles = [] } = preset;
+
+  const isNative = ['ios', 'android'].includes(haste.defaultPlatform);
+
+  const setupFilePlatform = isNative ? 'native' : 'web';
+
+  if (isNative) {
+    return {
+      ...preset,
+      setupFilesAfterEnv: [
+        ...setupFilesAfterEnv,
+        require.resolve(`./setupEnzyme.${setupFilePlatform}.js`),
```

Comment:
write out the path so it's more greppable, especially since it is a file path

```suggestion
        require.resolve(`./setupEnzyme.native.js`),
```